legend-engine-xts-sql/legend-engine-xt-sql-postgres-server/pom.xml

```diff
@@ -410,6 +423,21 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>com.google.guava</groupId>
```

Comment:
Can we add a comment here on why we are overriding the `guava` version from root pom?